modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -59,7 +60,24 @@ $ oc -n openshift-nmstate delete nmstate nmstate
 $ oc delete --all deployments --namespace=openshift-nmstate
 ----
 
-. Delete all the custom resource definition (CRD), such as `nmstates`, that exist in the `nmstate.io` namespace by running the following commands:
+. After you deleted the `nmstate` CR, remove the `nmstate-console-plugin` plugin resource from the `nmstate` CR so that `nmstate-console-plugin` does not persist.
```

Comment:
```suggestion
. After you deleted the `nmstate` CR, remove the `nmstate-console-plugin` console plugin name from the `console.operator.openshift.io/cluster` CR.

Reply:
Also, this step describes the following next steps, it's not a step on its own.

Reply:
:heavy_check_mark: